recipe/post-link.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
-
-"%PREFIX%\Scripts\jupyter.exe" nbextensions_configurator enable --sys-prefix
-if errorlevel 1 exit 1
```

Comment:
do we not need to keep the `if errorlevel 1 exit 1` to ensure a sensible exit status? cmd is all a bit cargo-culture to me, so I'm not really sure...